### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by calebk1**
This PR makes style updates to all the documents in the "Getting Started" section.

### CODE REVIEW

File: getting-started/configure-datastreams.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-To create an Datastream, your organization must be provisioned for this feature in Adobe Experience Platform Launch. Please contact your Adobe Customer Success Manager \(CSM\) to be added to the _allow list_.
+To create a datastream, your organization must be provisioned for this feature in Adobe Experience Platform Launch. Please contact your Adobe Customer Success Manager (CSM) to be added to the _allow list_.
 {% endhint %}
 
 {% hint style="info" %}
 The Datastreams tool is available to customers on the _allow list_ regardless whether they use Experience Platform Launch for web tag management or to manage mobile app configuration.
 
-Users will require _Develop_ permissions in Experience Platform Launch. See the [User Permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html) article in the Experience Platform Launch documentation for more details.
+Users will require _Develop_ permissions in Experience Platform Launch. See the [user permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html) article in the Experience Platform Launch documentation for more details.
```
Comment (emdobrin) [inline getting-started/configure-datastreams.md:12]: Should this match the title of the article, like `[User permissions](link)` ?

File: getting-started/configure-datastreams.md [hunk 2]
```diff
@@ -19,24 +19,24 @@ A datastream is a configured instance of the Edge Network that contains settings
...
 
 1. In the browser, navigate to [Adobe Experience Platform Launch](https://experience.adobe.com/launch) and login with your credentials.
-2. From the left panel, choose **Datatstreams** from the dropdown \(instead of **Client Side**\).
-3. Click **New Datatstream** \(located on top right\)
+2. From the left panel, choose **Datastreams** from the dropdown (instead of **Client Side**).
+3. Click **New Datastream** (located on top right)
 4. Set a name for the configuration and click **Save**.
 
-![Creating an Datastream in Adobe Experience Platform Launch](../.gitbook/assets/screen-shot-2021-02-02-at-12.42.57-pm%20%281%29.png)
+![Creating a datastream in Adobe Experience Platform Launch](../.gitbook/assets/screen-shot-2021-02-02-at-12.42.57-pm%20%281%29.png)
```
Comment (emdobrin) [inline getting-started/configure-datastreams.md:26]: Can you also update the screenshot to match the new Launch UI and the description?

File: getting-started/configure-datastreams.md [hunk 3]
```diff
@@ -19,24 +19,24 @@ A datastream is a configured instance of the Edge Network that contains settings
...
 2. Once you have made your selections click `Save`.
 
-![Configuring your Datatstream in Adobe Experience Platform Launch](../.gitbook/assets/aep-edge-config.png)
+![Configuring your datastream in Adobe Experience Platform Launch](../.gitbook/assets/aep-edge-config.png)
 
-In the resulting summary screen, you will see that three environments have been created for your Datatstream. If needed, each environment can be edited individually with different configuration parameters.
+In the resulting summary screen, you will see that three environments have been created for your datastream. If needed, each environment can be edited individually with different configuration parameters.
 
-Make note of the listed, Launch Datatstream **Environment ID**s, as these values will be referenced when you setup the mobile property in the next step.
+Make note of the listed, Launch Datastream Environment IDs, as these values will be referenced when you setup the mobile property in the next step.
```
Comment (emdobrin) [inline getting-started/configure-datastreams.md:37]: Let's remove this - these IDs are not used in the next step.

File: getting-started/configure-datastreams.md [hunk 4]
```diff
@@ -19,24 +19,24 @@ A datastream is a configured instance of the Edge Network that contains settings
...
 
 ![Environment IDs for your Datatstream](../.gitbook/assets/screen-shot-2021-02-02-at-12.45.55-pm.png)
 
-For additional information on datatstreams in Experience Platform Launch, see the [reference guide](https://experienceleague.adobe.com/docs/experience-platform/edge/fundamentals/edge-configuration.html).
+For additional information on datastreams in Experience Platform Launch, see the [reference guide](https://experienceleague.adobe.com/docs/experience-platform/edge/fundamentals/edge-configuration.html).
```
Comment (emdobrin) [inline getting-started/configure-datastreams.md:41]: let's update to:
```
see the [Configuring a Datastream](https://experienceleague.adobe.com/docs/experience-platform/edge/fundamentals/datastreams.html) guide.
```

File: getting-started/create-a-mobile-property.md [hunk 5]
```diff
@@ -10,11 +10,11 @@ Before you can set up your mobile property, complete the following prerequisites
 
 Launch is fully integrated with your Adobe ID. User permissions are managed through the Admin Console with other Adobe products and solutions from the Creative Cloud, Document Cloud, and Experience Cloud.
 
-For detailed instructions on how to create groups and add users for Launch, see [User permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html).
+For detailed instructions on how to create groups and add users for Launch, see [user permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html).
```
Comment (emdobrin) [inline getting-started/create-a-mobile-property.md:13]: Same question here, should it be `[User permissions](link)` ?

File: getting-started/create-a-mobile-property.md [hunk 6]
```diff
@@ -10,11 +10,11 @@ Before you can set up your mobile property, complete the following prerequisites
...
 
 ### Log in to Experience Platform Launch
 
-After Experience Platform Launch rights have been added to your Adobe ID, log in to Launch. You can do this by going to [https://launch.adobe.com](https://launch.adobe.com) or by logging in to the Experience Cloud \([https://experiencecloud.adobe.com](https://experiencecloud.adobe.com)\), navigating to the Activation page, and clicking on Launch.
+After Experience Platform Launch rights have been added to your Adobe ID, log in to Launch. You can do this by going to [https://launch.adobe.com](https://launch.adobe.com) or by logging in to the Experience Cloud ([https://experiencecloud.adobe.com](https://experiencecloud.adobe.com)), navigating to the Activation page, and selecting Launch.
```
Comment (emdobrin) [inline getting-started/create-a-mobile-property.md:17]: should `https://launch.adobe.com` be updated to `https://experience.adobe.com/#/data-collection` ?

File: getting-started/enable-debug-logging.md [hunk 7]
```diff
@@ -149,7 +145,7 @@ With the `onResume` function, start Lifecycle data collection:
 ```
 
 {% hint style="info" %}
-Setting the application is only necessary on activities that are entry points for your application. However, setting the application on each Activity has no negative impact and ensures that the SDK always has the necessary reference to your application. We recommend that you call `setApplication`in each of your activities.
+Setting the application is only necessary on activities that are entry points for your application. However, setting the application on each activity has no negative impact and ensures that the SDK always has the necessary reference to your application. As a result, you should call `setApplication` on each of your activities.
```
Comment (emdobrin) [inline getting-started/enable-debug-logging.md:148]: Here Activity is the name of a java class, so we should keep it `Activity`

**Review by emdobrin — COMMENTED** [review]
@calebk1 left few comments bellow. 
As a general question, for links can we alter the link title when referencing an external link in our docs? We used to use the same title in our documentation as the article had.

File: getting-started/configure-datastreams.md (hunk 1 above)
Comment (calebk1) [reply getting-started/configure-datastreams.md:12]: For the Platform docs, we've been moving away from just exactly matching the title name, in favour of having the names more contextualized within the document, which is why I've changed it to be like that.

File: getting-started/configure-datastreams.md (hunk 2 above)
Comment (calebk1) [reply getting-started/configure-datastreams.md:26]: Hmm, I don't think I have access to the Launch UI, unfortunately - if you know someone who can take the screenshot for me, I'd be happy to update it!

File: getting-started/create-a-mobile-property.md [hunk 8]
```diff
@@ -77,24 +79,24 @@ To deploy your configuration to a development environment for testing:
...
+7. On the library card, click the ellipsis (...) to see a dropdown list.
 8. Select **Submit for Approval** and then **Submit**.
 
-The library of changes are then published to the Development environment and the library is displayed under the **Submitted** section of the publishing workflow.
+The library of changes are then published to the development environment and the library is displayed under the **Submitted** section of the publishing workflow.
 
 {% hint style="info" %}
-Testing can be done using the configuration in the Development environment. The library can later be deployed to the **Staging** and **Production** environments by using the rest of the publishing workflow. For more information, see [Publishing](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html).
+Testing can be done using the configuration in the development environment. The library can later be deployed to the **Staging** and **Production** environments by using the rest of the publishing workflow. For more information, see the documentation on [publishing in Launch](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html).
```
Comment (ahmyr) [inline getting-started/create-a-mobile-property.md:88]: @calebk1, Development is a proper name here. We have three environments created by default called "Development", "Staging" and "Production". They can, however, create additional development environments and call them something other than "Development". With that context, not sure if we capitalize "Development" here or not?

File: getting-started/validate.md [hunk 9]
```diff
@@ -4,33 +4,35 @@ As you begin to add solution APIs to your mobile implementation, you are going t
...
 
-![Project Griffon Analytics View](../.gitbook/assets/GriffonAnalytics.png)
+![Project Griffon analytics view](../.gitbook/assets/GriffonAnalytics.png)
 
-## Places Service View
+## Places Service view
 
-Testing and validating actions based on geofence data can be incredibly difficult and time consuming. Project Griffon helps ease the pain in testing point of interest entries and exits. The Places event list provides a focused view showing all events related to Places Service, including user authorization level granted and requests for nearby points of interest \(POIs\). While a device is connected to an active Griffon session the map view will show a timeline of POI entries and exits. If you would like to test actions or experiences triggered by geofence entries and exits, the map view will allow you to spoof or simulate your location by clicking on any area within the map.
+Project Griffon simplifies testing point-of-interest entries and exits. The Places event list provides a focused view showing all events related to Places Service, including user authorization level granted and requests for nearby points of interest (POIs).
```
Comment (ahmyr) [inline getting-started/validate.md:21]: @calebk1, we have hyphenated "points of interest" in the first sentence, but not the second. Both are referring to the same thing. We call them "points of interest" or "POIs".

File: getting-started/create-a-mobile-property.md (hunk 8 above)
Comment (calebk1) [reply getting-started/create-a-mobile-property.md:88]: Ah ok, yup that makes sense. I'll re-capitalize it then.

File: getting-started/validate.md (hunk 9 above)
Comment (calebk1) [reply getting-started/validate.md:21]: Yeah, it should be all un-hyphenated. I'll fix that!

File: getting-started/create-a-mobile-property.md [hunk 10]
```diff
@@ -77,24 +79,24 @@ To deploy your configuration to a development environment for testing:
 
    The library builds and is displayed under the **Development** section of the publishing workflow.
 
-7. On the library card, click _..._ to see a drop-down list.
+7. On the library card, click the ellipsis (...) to see a dropdown list.
 8. Select **Submit for Approval** and then **Submit**.
 
-The library of changes are then published to the Development environment and the library is displayed under the **Submitted** section of the publishing workflow.
+The library of changes are then published to the development environment and the library is displayed under the **Submitted** section of the publishing workflow.
```
Comment (emdobrin) [inline getting-started/create-a-mobile-property.md:85]: Development environment

File: getting-started/enable-debug-logging.md [hunk 11]
```diff
@@ -267,13 +265,18 @@ ACPCore.lifecyclePause();
...
+{% endhint %}
+
 {% endtab %}
 
 {% tab title="Cordova" %}
 ## Cordova
 
-> Note: We recommend implementing Lifecycle in native [Android and iOS code](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
+{% hint style="info" %}
+You need to implement Lifecycle in native Android and iOS code. For more information on implementing, please read the [Lifecycle documentation](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
```
Comment (emdobrin) [inline getting-started/enable-debug-logging.md:278]: the new link is https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle

**Review by emdobrin — APPROVED** [review]

File: getting-started/configure-datastreams.md (hunk 2 above)
Comment (emdobrin) [reply getting-started/configure-datastreams.md:26]: 👍  provided couple screenshots to you

**Pull request merged**